base/ltfssbas.dtx

```diff
@@ -1184,18 +1184,25 @@
 % \changes{v1.2g}{1990/02/16}{\cs{@nomath} added.}
 % \changes{v2.1a}{1994/01/17}{New math font setup}
 % \changes{v3.2l}{2023/04/01}{Reset frozen mathversion gh/1028}
+% \changes{v3.2o}{2024/11/30}{New appoach to
+%    \texttt{localmathalphabets} (gh/1101)}
 %    \begin{macrocode}
 \DeclareRobustCommand\mathversion[1]
          {\@nomath\mathversion
           \expandafter\ifx\csname mv@#1\endcsname\relax
           \@latex@error{Math version `#1' is not defined}\@eha\else
 %    \end{macrocode}
-%  If there has been a frozen math version reset locally. See GH 1028.
+%    If there has been a frozen math version reset unconditionally to
+%    it, so that in case of bold inside normal math we always restart.
+%    This probably fails if you have several levels of nesting with
+%    different math versions so perhaps not a good idea. Maybe in
+%    addition one would need to look at the \cs{@math@level} and do
```

Comment:
I think in the code below you are now testing math@level =1 so "Maybe is addition" not really applying.

Reply:
yeah a leftover not updated after improving the implementation